tools/cmux-home/scripts/smoke.py

```
@@ -0,0 +1,323 @@
+        "unknown option",
+        "unrecognized option",
+        "flag provided but not defined",
+        "unexpected argument",
+    ]
+    return any(marker in text for marker in markers)
+
+
+def run_once(label: str, command: list[str], args: list[str]) -> subprocess.CompletedProcess[str]:
+    full_command = command + args
+    print(f"RUN {label}: {' '.join(shlex.quote(part) for part in full_command)}")
+    result = subprocess.run(full_command, text=True, capture_output=True)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)